### ISSUE TITLE
Protocol 1.0 not supported?

### ISSUE THREAD

**Issue by andreagavazzi** [yoshito-n-students/layered_hardware_dynamixel#4]
It seems that only protocol 2.0 is supported as operating mode names for protocol 1.0 are only joint_mode and wheel_mode.
When using Dynamixel on protocol 1.0 (like AX_18A) this error happens

OperatingModeBase::readItem(): Failed to read control table item 'Present_Velocity' of 'head_1_motor' (id: 1): [DynamixelTool] Can't find Item

I think because in protocol 1.0 present_velocity is called present_speed.
Any chance to have a "joint_mode" operating mode?

**Comment by yoshito-okada** [comment]
Yes, layered_hardware_dynamixel only supports protocol 2.0 because I hardly see dynamixels with 1.0.  Unfortunately I have no plan to support 1.0 for now, but a PR is always welcome.
